src/orchestration/dags/config/unified_pipeline.yaml

```
@@ -1,32 +1,32 @@
 ---
-release_name: '25.09'
+release_name: '25.12'
 
 # If `is_dev` is true:
 #   * the bucket used for the run will be `open-targets-pipeline-runs`
 #   * the prefix inside the bucket will be `run_name`
 #         which defaults to a timestamp if not provided.
 is_dev: true
-run_name: 'jferrer/test-edp-1'
+run_name: 'il/25.12-testrun-1'
 
 # Whether this will be a partner preview platform release (PPP)
 is_ppp: false
 
 # Versions for software and data
-pis_version: '25.2.3.dev.18'
-pts_version: '25.3.0-dev.11'
+pis_version: '25.2.3-dev.21'
+pts_version: '25.3.1-dev.6'
```

```suggestion
pts_version: '25.3.1-dev.10'
```

---

Orphanet XML reading

```suggestion
pts_version: '25.3.1-dev.11'
```